derive_builder/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.4.3] - 2017-04-11
```

Comment:
You can just add `## [Unreleased]` at the top in the future - at least that's how I do it. I've got that workflow from http://keepachangelog.com/. ;-)